### ISSUE TITLE
UI-kit: compobox and tree view should be keyboard accessible (arrows) 

### ISSUE THREAD

**Issue by SukuWc** [intechstudio/grid-uikit#32]
## Tree-view (used in profile cloud)
Sub-tree should be opened and closed using Space or Enter
Focus should be clearly visible when navigating

## ~Combobox (used mostly in actionblocks)~
~Allow open and close using Space or Enter~
~Use up and down arrows to navigate options~
~Focus should be clearly visible when navigating~

## Add Folder and ConfigCard component to UI-kit
